ext/opcache/tests/jit/dim_assign_001.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+JIT - Assigning to arrays using string key which parses to an integer
+--SKIPIF--
+<?php require_once('skipif.inc'); ?>
+--INI--
+opcache.enable_cli=1
+opcache.jit_hot_func=1
+opcache.jit_buffer_size=1M
```

Comment:
It's better to just leave off explicit ini settings, which means that it will get tested in different modes (no opcache, no jit, function jit, tracing jit).

Reply:
Hmm. But all the existing tests in `ext/opcache/tests/jit` use explicit INI settings.

Reply:
Right. Personally I wouldn't even put this inside ext/opcache/tests/jit, but into Zend/tests. While it tests a JIT bug, the code itself has no dependence on the JIT.

Reply:
Oh, OK. Sounds good.